src/test/TestUtils/Command.cs

```diff
@@ -28,7 +26,12 @@ public class Command
         private bool _running = false;
         private bool _quietBuildReporter = false;
 
-        public Process Process { get => _process; }
+        public Process Process { get; }
+
+        // Priority order of runnable suffixes to look for and run
+        public static readonly string[] RunnableSuffixes = RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
```

Comment:
Does this need to be public? I only see it used in this class.

Reply:
Good catch - thanks!